.github/workflows/main.yml

```diff
@@ -25,4 +25,10 @@ jobs:
         env:
           SCALA_VERSION: ${{ matrix.scala-version }}
           TEST_SET: ${{ matrix.test-set }}
-        run: ci/tests/run $TEST_SET $SCALA_VERSION
\ No newline at end of file
+        run: ci/tests/run $TEST_SET $SCALA_VERSION
+      - name: Upload screenshots
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: screenshots-${{ matrix.suite-name }}
```

Comment:
```
      matrix:
        scala-version: [ "2.12.10", "2.13.1" ]
        test-set: [ "probe", "scala" ]
```
this is the matrix, so suite-name is not even there, probably artifacts would overwrite one another